src/devices/Device.cpp

```diff
@@ -114,14 +114,14 @@ uint8_t Device::getNumberOfLeds() const {
 }
 
 void Device::packData() {
-	if (LEDs == oldLEDs) {
```

Comment:
override this on the adaligh controller (src/devices/Adalight/Adalight.cpp). this is used for other controllers as well to avoid sending data with nothing changed.

Reply:
I tried and after a full day couldn't figure out how so I resorted to this for it to work, I would be glad if you can show me how to do it.